docs/GitBranchSetup.md

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## Git branch setup
+
+This document describes how to set up your development environment and contribute changes to
+[accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows). This document assumes basic working knowledge
+with Git and related tools. We are providing instructions specific to this project.
+
+## Creating your own fork
+
+If you wish to contribute changes back to the [accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows)
+repository, start by creating your own [Fork](https://help.github.com/en/articles/fork-a-repo) of the repository. This will keep the
+number of branches on the main repository to a small
+count. In your own fork, you can create as many branches as you like.
+
+-   Navigate to [GitHub](https://github.com/) with a browser and log in to your GitHub account. For the sake of this document, let's assume your username is **adacat**.
+-   Navigate to the [accessibility-insights-windows](https://github.com/Microsoft/accessibility-insights-windows) repository in the same browser session.
+-   Click on the **Fork** button at the top right corner of the page.
+-   Create the fork under your account. Your GitHub profile should now show **accessibility-insights-windows** as one of your repositories.
+-   Create a folder on your device and clone your fork of the **accessibility-insights-windows** repository. e.g. `https://github.com/adacat/accessibility-insights-windows`. Notice how your GitHub username is in the repository location.
+
+```
+> git clone https://github.com/adacat/accessibility-insights-windows
```

Comment:
Should we actually have an AdaCat user/repo?

Reply:
@RobGallo we do have one but I didn't realize the docs spell it wrong. It should be ada-cat. You can see it [here](https://github.com/ada-cat). I will update.